### ISSUE TITLE
Partially entered multi-key keystroke exception when toolbar button doesn't have keyboard equivalent

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#682]
_[Trac time 20090606 191246Z]_ Related to #681: partially entered multi-key sequence followed by a toolbar button that doesn't have a keyboard equivalent results in:

```
Traceback (most recent call last):
  File "c:\Documents and Settings\
ob\Desktop\peppy-0.0.0\peppy\lib\multikey.py", line 1302, in OnMenu
    self.foundMenuAction(evt)
  File "c:\Documents and Settings\
ob\Desktop\peppy-0.0.0\peppy\lib\multikey.py", line 1343, in foundMenuAction
    keystroke = self.findKeystrokeFromMenuId(eid)
  File "c:\Documents and Settings\
ob\Desktop\peppy-0.0.0\peppy\lib\multikey.py", line 1313, in findKeystrokeFromMenuId
    return self.findKeystrokeFromMenuAction(action)
  File "c:\Documents and Settings\
ob\Desktop\peppy-0.0.0\peppy\lib\multikey.py", line 1333, in findKeystrokeFromMenuAction
    raise KeyError
KeyError
```

**Comment by robmcmullen** [comment]
_[Trac time 20090608 035706Z]_ [old svn rev2109] Fixed #681 and #682: toolbar press now forces cancellation of in-progress multi-key combination
- processes the toolbar button instead

**Comment by robmcmullen** [comment]
_[Trac time 20090607 160841Z]_ Not just MSW.
